Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -118,18 +128,24 @@ var DatePickerIOS = React.createClass({
 
   render: function() {
     var props = this.props;
+    let minimumDate = props.minimumDate;
```

Comment:
Does this need to be inside render?  I think it can be set above alongside `propsTimeStamp`.